skimage/morphology/_util.py

```diff
@@ -51,35 +54,206 @@ def _validate_connectivity(image_dim, connectivity, offset):
     return c_connectivity, offset
 
 
-def _offsets_to_raveled_neighbors(image_shape, structure, center, order='C'):
+def _offsets_to_raveled_neighbors(image_shape, selem, center, order='C'):
```

Comment:
Thanks for the change for a better consistency. We should definitely make this list of variable names in the contributing notes :-).

Reply:
Sorry, can I be the annoying objector here?

I think originally we stopped using selem because it is a very obscure term for people who are not familiar with image processing, and morphology in particular. Structure is a bit better, but imho "footprint" or "neighborhood" are the clearest. What do we think? Consistency is great, but let's think about what the best syntax is going forward, and be consistent with that.

Reply:
I have no clear preference concerning the best argument name for this, I just want to draw attention to the existence of the `skimage.morphology.selem` module.

Reply:
@jni I like `neighborhood`. But think this should be a separate PR. In any case I think `selem` is better than the old `structure` because of the afore mentioned consistency. So until we change this everywhere I'd leave it as `selem`. 

Not a strong opinion though. I'll change it if you find a silent consensus.

Reply:
This function is private. The choice is not critical at the end. :)